packages/core/contracts-ovm/insured-bridge/implementation/AVM_BridgeDepositBox.sol

```diff
@@ -86,7 +106,14 @@ contract AVM_BridgeDepositBox is BridgeDepositBox {
 
         whitelistedTokens[l2Token].lastBridgeTime = uint64(getCurrentTime());
 
-        // TODO: Add Arbitrum specific cross-chain token sending over the canonical bridge.
+        StandardBridgeLike(l2GatewayRouter).outboundTransfer(
```

Comment:
Is the `l2GatewayRouter` going to be a hardcoded address in production? Do you think we should mimic the Optimism style and add its address to some library so that the constructor interface exactly resembles the OVM deposit box? I think this is a small concern so there is probably nothing to do but curious what you think.

Reply:
ye, I dont think it will change in production. on their current deployment it is a transparent proxy in front of the router so even if they upgrade it the address will remain the same. This can be set once during construction.

Regarding the OVM setup of using a lib for this that would be nice but I did not find an Arbitrum equivalent file to import that had this address.

Reply:
Ok sounds good to leave as is